projects/core/modules/notifications/notification-alert/notification-alert.template.html

```diff
@@ -4,13 +4,17 @@
     [hasIcon]="safeItem.hasIcon"
     (close)="closeNotification()"
 >
-    <label
-        *ngIf="safeItem.label"
-        automation-id="tui-notification-alert__heading"
-        class="heading"
-    >
-        {{safeItem.label}}
-    </label>
+    <span polymorpheus-outlet [content]="label" [context]="context">
+        <ng-template let-label>
+            <label
+                *ngIf="label"
```

Comment:
I think we should just add `polymorpheus-outlet` to this `label`. That is, if we end up expanding to polymorpheus here.

Reply:
Done